apps/developer-hub/src/components/CopyAddress/index.tsx

```diff
@@ -0,0 +1,24 @@
+"use client";
+
+import { CopyButton } from "@pythnetwork/component-library/CopyButton";
+import { Link } from "@pythnetwork/component-library/Link";
+
+import TruncateToMiddle from "../TruncateToMiddle";
+import styles from "./index.module.scss";
+
+const CopyAddress = ({ address, url }: { address: string; url?: string }) => {
+  return url ? (
+    <div className={styles.address}>
+      <Link href={url} target="_blank" rel="noreferrer">
+        <TruncateToMiddle text={address} />
+      </Link>
+      <CopyButton text={address} iconOnly />
+    </div>
+  ) : (
+    <CopyButton text={address}>
+      <TruncateToMiddle text={address} />
+    </CopyButton>
+  );
+};
+
+export default CopyAddress;
```

Comment:
did we decide we don't want to extract & reuse the same component from entropy-explorer for some reason?  I think it would be better to share the same component given this is effectively identical functionality

Reply:
Oh I just saw your comment about their being a Copy component in the library, so that's what I used. I'll look at the explorer components and work on getting it extracted.

Reply:
Yeah that's exactly what I had in mind, the only coupling in that component to `EntropyDeployments` is to figure out the href value, so just leave that part in entropy-explorer and extract the rest of the component & styling to the component library